packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -154,6 +154,20 @@ export interface DropdownProps
   /** A message to be displayed in the list when the dropdown has no items. */
   noResultsMessage?: ShorthandValue<ListItemProps>
 
+  /**
+   * Called when the dropdown's selected items index change.
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props and the new selected value(s).
+   */
+  onActiveSelectedIndexChange?: ComponentEventHandler<DropdownProps>
```

Comment:
```suggestion
  onActiveSelectedIndexChange?: ComponentEventHandler<DropdownProps>
```

It's aligned, but name looks really strange 💣

Reply:
:(